TechtonicaModLoader/MVVM/ViewModels/Settings/SettingViewModelHelper.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Microsoft.Win32;
+using System.Diagnostics;
+using System.IO;
+using TechtonicaModLoader.Services;
+using TechtonicaModLoader.Stores;
+
+namespace TechtonicaModLoader.MVVM.ViewModels.Settings
+{
+    internal class SettingViewModelHelper 
+    {
+        // Properties
+        public List<SettingBase> SettingViewModels { get; } = [];
+
+        public string DefaultCategory { get; }
+
+        internal SettingViewModelHelper(UserSettings userSettings, IDialogService dialogService, Action settingChanged, string defaultCategory) {
+
+            DefaultCategory = defaultCategory;
+
+            // General
+
+            SettingViewModels.Add(new Setting<bool>(
```

Comment:
Nothing needs to know about these ViewModels other than as a part of the collection so just create & add them.   Anything else that needs as setting should get it from IUserSettings.